Create 2021.23-24 release notes (#333)
Crunch-io/ta-da: content/release-notes/2021-07-02.md

+++
date = "2021-07-02T10:00:00-07:00"
publishdate = "2021-07-02T10:00:00-07:00"
draft = false
title = "Release 2021.23-24"
images = ["https://crunch.io/img/logo-1200x630.png"]
+++

### Crunch web application

> - Fixed an issue that prevented the "Set comparison" button for hypothesis testing of tables from working properly when the option to show only categories is selected.

Reviewer: ```suggestion
- Fixed an issue that prevented the “Set comparison” button for hypothesis testing of tables from working properly when the option to show only categories is selected.
```

- Added the ability to include subtotals to multiple response variables (in early access for editors).

> - Added export support for numeric array variable summary cards that are saved to a deck, in use cases when a weight has been applied to a dataset.

Reviewer: ```suggestion
- Added support for deck export of numeric array variable cards.
```

- Made the following fixes:
    - a multitable consisting of categorical variables now loads properly.

>     - a multitable consisting of a multiple response variable now loads properly when the row variable was a numeric.

Reviewer: ```suggestion
    - a multitable with a multiple response column variable correctly displays means of numeric row variables.
```

>     - differences now display in the 'Edit' preview for a categorical array analysis.

Reviewer: ```suggestion
    - differences now display in the ‘Edit slide/tile’ preview for categorical arrays.
```

>     - "hides" are now working for the columns variable in the 'Edit Categories' tab.

Reviewer: ```suggestion
    - hiding column categories works in the ‘Edit Categories’ tab.
```

- Added more support for both Confirmit and Decipher integrations (see help desk articles below for more information).

> - Removed the faulty value in the intersection between the weighted/unweighted 'rows and mean' column.
> - Fixed the "x" and "remove all" buttons in the 'dashboard groups' panel.

Reviewer: ```suggestion
- Removed a meaningless value in the intersection between the weighted/unweighted ‘rows and mean’ column of exports.
- Fixed multiple issues in dashboard ‘groups’ panel that prevented removing or adding them.
```

- Fixed rare behavior that causes some export settings to fail.

### Crunch Automation

- Made more speed improvements to the [UNSET NUMERIC VALUE command](https://help.crunch.io/hc/en-us/articles/360049589551-UNSET-NUMERIC-VALUE-command) when using numerous variables.
- Added support to the [CHANGE command](https://help.crunch.io/hc/en-us/articles/360042038132-CHANGE-command) (`LABELS`)  so that it can use functions alongside the explicit list of labels.

### Help desk and website improvements

> - Updated the [Category Subtotals article](https://help.crunch.io/hc/en-us/articles/360050364772-Category-Subtotals) with new information about how to add subtotals (a.k.a. "nets") to your multiple response variables.

Reviewer: ```suggestion
- Updated the [Category Subtotals](https://help.crunch.io/hc/en-us/articles/360050364772-Category-Subtotals) article with new information about how to add subtotals to multiple response variables.
```

- Created a new article that describes how [metadata maps from Confirmit](https://help.crunch.io/hc/en-us/articles/4403966631309-How-your-survey-metadata-maps-from-Confirmit-into-Crunch) into Crunch.
- Added new information about how [Decipher logic elements ("Loops")](https://help.crunch.io/hc/en-us/articles/360062142812-How-your-survey-metadata-maps-from-Decipher-into-Crunch#toc4) map into Crunch.

> - Updated the [chart](https://help.crunch.io/hc/en-us/articles/360061204751-Crunch-importers-overview) in the Crunch importers overview article.

Reviewer: ```suggestion
- Updated the [feature matrix](https://help.crunch.io/hc/en-us/articles/360061204751-Crunch-importers-overview) in the Crunch importers overview article.
```
